src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
+const DEFAULT_HEALTH = 100;
+const DAMAGE = 50;
+
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name, health = DEFAULT_HEALTH) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
```

Comment:
Consider adding a method in the Animal class to handle the removal of dead animals from the alive array. This will encapsulate the logic for removing dead animals and can be called from the Carnivore's bite method.